src/table/hooks/use-keyboard-active-listener.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+import { FocusTypes } from '../constants';
+import { useContextSelector, TableContext } from '../context';
+import { updateFocus } from '../utils/accessibility-utils';
+import { getCellElement } from '../utils/get-element-utils';
+
+/**
+ * When nebula handles keyboard navigation (keyboard.enabled === true)
+ * and keyboard.active changes, either focus the first cell or blur the focusedCellCoord cell
+ * when keyboard.focus() runs, keyboard.active is set to true
+ * when keyboard.blur() runs, keyboard.active is set to false
+ */
+const useKeyboardActiveListener = () => {
+  const { rootElement, keyboard } = useContextSelector(TableContext, (value) => value.baseProps);
+  const focusedCellCoord = useContextSelector(TableContext, (value) => value.focusedCellCoord);
```

Comment:
I don't know, I thought this was cleaner. Might as well pass them from tableWrapper